elixir/apps/web/lib/web/live/settings/dns.ex

```diff
@@ -1,7 +1,113 @@
 defmodule Web.Settings.DNS do
   use Web, :live_view
 
+  alias Domain.Config
+
+  defp remove_errors(changeset, field, message) do
+    errors =
+      changeset.errors
+      |> Enum.filter(fn err ->
+        case err do
+          {^field, {msg, _}} ->
+            if msg == message, do: false, else: true
+
+          {_, _} ->
+            true
+        end
+      end)
+
+    %{changeset | errors: errors}
+  end
+
+  defp filter_errors(changeset) do
+    filtered_cs =
+      changeset
+      |> remove_errors(:clients_upstream_dns, "address can't be blank")
+
+    filtered_dns_cs =
+      filtered_cs.changes.clients_upstream_dns
```

Comment:
There might be no changes and this code will crash.

Reply:
Ah, yeah, that makes sense.  Thanks for the suggestion!  Will get this updated.